sklearn/cluster/tests/test_optics.py

```diff
@@ -129,7 +129,7 @@ def test_extract_xi():
 def test_cluster_hierarchy_():
     n_points_per_cluster = 100
```

Comment:
you can also set the seed here to be on the safe side.

Reply:
hmm, forgot my own comment o(╥﹏╥)o